ote_cli/ote_cli/registry/registry.py

```diff
@@ -52,3 +56,14 @@ def get(self, name):
 
     def __repr__(self):
         return yaml.dump(self.templates)
+
+
+def find_and_parse_model_template(path_or_name):
+    if os.path.exists(path_or_name):
+        return parse_model_template(path_or_name)
+    template = [template for template in Registry('.').templates if template['name'] == path_or_name]
+    if template:
```

Comment:
What if multiple matching templates are found?

Reply:
It must not be so, right?

Reply:
If name == model_template_id, then yes. ID is unique, but not the name.

Reply:
ok, then path and id can be used only.